### PR TITLE
 	Re-activate and Update Yahoo_Japan.xml 

### PR DESCRIPTION
**Pull request by cschanaj**
This is ready for review now. thanks.

### CODE REVIEW

**Comment by jeremyn** [comment]
You can [run tests locally](https://github.com/EFForg/https-everywhere/blob/master/ruleset-testing.md#testing-and-continuous-build) by the way, you don't need to rely on Travis, if that's why you made a "work-in-progress" pull request.

**Comment by cschanaj** [comment]
@jeremyn Thank you for your advice. I had found that lots of comment are outdated and I would like to update them. I wanted to passed them to my bot but it require manual editing and sorting to fit the input format. In particular, I would like a make a script to sort the sub-domains from the comment to the desired order automatically. It was late at night so I made it _work-in-progress_. 

Sorry for the inconvenience incurred. 

P.S. Just in case if anyone is interested in the script, see  https://github.com/cschanaj/dsort

**Comment by jeremyn** [comment]
Another FYI, the [Sublist3r](https://github.com/aboul3la/Sublist3r) application that we recommend in the [contributing guidelines](https://github.com/EFForg/https-everywhere/blob/5609071840e4016c5ed2859d0e99ffb328517b3c/CONTRIBUTING.md#enumerating-subdomains) sorts the domains the way we like (HTTPS Everywhere contributors have also contributed to Sublist3r). You may want to use that instead of your custom stuff, for instance [here](https://github.com/aboul3la/Sublist3r/blob/f8398ecaa58f3fe29dae57b9e9a368465472e598/sublist3r.py#L111-L134).

**Comment by cschanaj** [comment]
I usually use `Sublist3r` as recommended when the number of sub-domains is small. However, for some organizational domain like universities, the output list is simply overwhelming even with automated ruleset generation tools available (and many sub-domains have unknown usage). For example, I remember that I got more than 2k sub-domains using `Sublist3r` when I was creating a ruleset for `www.ust.hk`, it is difficult to decide which sub-domains should be included in a ruleset.

As for this time, `yahoo.co.jp` has 1.66k sub-domains. I really do not want to go through the `Sublist3r` output. Therefore  I manually construct the _yahoo-japan-sub-domains.txt_ from the comment (manually) and crawl sub-domains from the homepage as the source. AFAIK `Sublist3r` doesn't support sorting input from external source, so I made my own one (for personal use only).

Anyway, thanks!!

**Comment by cschanaj** [comment]
#10545

**Comment by koops76** [comment]
@J0WI ping

**Comment by J0WI** [comment]
Pong.
I think @jeremyn is doing the review here, so let's just wait for his answer.

**Comment by jeremyn** [comment]
I'm not particularly involved here.

File: src/chrome/content/rules/Yahoo_Japan.xml [hunk 1]
```diff
@@ -193,78 +72,119 @@ Fetch error: http://lot.sample.yahoo.co.jp/ => https://lot.sample.yahoo.co.jp/:
...
+
+	<target host="finance.yahoo.co.jp" />
+	<target host="fortune.yahoo.co.jp" />
+
+	<target host="games.yahoo.co.jp" />
+	<target host="geocities.yahoo.co.jp" />
+	<target host="gyao.yahoo.co.jp" />
+
+	<target host="headlines.yahoo.co.jp" />
+	<target host="hr.yahoo.co.jp" />
```
Comment (J0WI) [inline src/chrome/content/rules/Yahoo_Japan.xml]: id.yahoo.co.jp

File: src/chrome/content/rules/Yahoo_Japan.xml [hunk 2]
```diff
@@ -193,78 +72,119 @@ Fetch error: http://lot.sample.yahoo.co.jp/ => https://lot.sample.yahoo.co.jp/:
...
+	<target host="realestate.yahoo.co.jp" />
+
 	<target host="s.sample.yahoo.co.jp" />
+	<target host="search.yahoo.co.jp" />
 	<target host="custom.search.yahoo.co.jp" />
-	<target host="auctions.yahoo.co.jp" />
-	<target host="chiebukuro.yahoo.co.jp" />
-	<target host="detail.chiebukuro.yahoo.co.jp" />
-	<!--	404
-	target host="odhistory.shopping.yahoo.co.jp" /-->
```
Comment (J0WI) [inline src/chrome/content/rules/Yahoo_Japan.xml]: Please keep this documented.

File: src/chrome/content/rules/Yahoo_Japan.xml [hunk 3]
```diff
@@ -193,78 +72,119 @@ Fetch error: http://lot.sample.yahoo.co.jp/ => https://lot.sample.yahoo.co.jp/:
...
+	<target host="list.chiebukuro.yahoo.co.jp" />
+	<target host="my.chiebukuro.yahoo.co.jp" />
+	<target host="note.chiebukuro.yahoo.co.jp" />
+	<target host="creative-award.yahoo.co.jp" />
+	<target host="crowdsourcing.yahoo.co.jp" />
+
+	<target host="docs.yahoo.co.jp" />
+
 	<target host="biz-form.ec.yahoo.co.jp" />
-	<!--target host="help.yahoo.co.jp" /-->
```
Comment (J0WI) [inline src/chrome/content/rules/Yahoo_Japan.xml]: Please keep this host documented

**Comment by cschanaj** [comment]
@J0WI thanks!!

**Pull request merged**